test/test_spellchecking.py

```diff
@@ -51,7 +51,7 @@ def test_spellchecking():
     result = spell.spell_check(text, 'Description({}):')
     assert len(result) == 2
     assert result['tihs'].startswith('Description(en_US): tihs -> ')
-    assert get_suggestions(result['tihs']) == ['hits', 'this', 'ties']
```

Comment:
Nothing against anything, but shouldn’t this be a separate PR?

Reply:
It's a different commit, good enough for me.

Reply:
ok